mim/utils/default.py

```diff
@@ -23,9 +23,15 @@
     'mmocr': 'mmocr',
     'mmgen': 'mmgeneration',
 }
-PKG_ALIAS = {
+PKG2MODULENAME = {
```

Comment:
How about simply using PKG2MODULE and MODULE2PKG

Reply:
yet, agree.